helmfile/values/opensearch/curator.yaml.gotmpl

```diff
@@ -0,0 +1,24 @@
+opensearch:
+  accountSecret: opensearch-curator-user
+  clusterEndpoint: {{ .Values.opensearch.clusterName }}-master:9200
+
+startingDeadlineSeconds: 600
```

Comment:
Maybe not part of the issue, but I would like to move this to the config file. Objections for that?

Reply:
Yes, basically the same as this https://github.com/elastisys/compliantkubernetes-apps/blob/54f1d1ab2434ebc4b19c163c373002e324623d66/helmfile/values/opensearch/slm.yaml.gotmpl#L9

Reply:
Added it, I also looked through and made the same across all cronjobs.